src/app/home/home.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="wrapper">
-	<img src="{{headerImageUrl}}" alt="Loklak Cow" width="auto" height="120">
+	<img src="{{headerImageUrl}}" alt="loklak Search - Distributed Social Media Message Search Engine" title="loklak Search - Distributed Social Media Message Search Engine" width="auto" height="120">
```

Comment:
I think the both width and height should be specified before loading instead of auto.